go.mod

```diff
@@ -1,44 +1,42 @@
 module github.com/ollama/ollama
 
-go 1.22
-
-toolchain go1.22.0
```

Comment:
Was this `toolchain` removed intentionally?

Reply:
add it back